ql/src/java/org/apache/hadoop/hive/ql/exec/ExplainTask.java

```diff
@@ -541,12 +606,12 @@ JSONObject collectAuthRelatedEntities(PrintStream out, ExplainWork work)
     if (delegate != null) {
       Class itface = SessionState.get().getAuthorizerInterface();
       Object authorizer = AuthorizationFactory.create(delegate, itface,
-          new AuthorizationFactory.AuthorizationExceptionHandler() {
-            @Override
-            public void exception(Exception exception) {
-              exceptions.add(exception.getMessage());
-            }
-          });
+                                                      new AuthorizationFactory.AuthorizationExceptionHandler() {
+                                                        @Override
```

Comment:
there are lots of pure indentation changes in this patch - are we using the same formatter settings?
`dev-support/eclipse-styles.xml`